deploy/operator/capi/deploy_capi_cluster.sh

```diff
@@ -166,5 +233,5 @@ if [ $(oc get baremetalhost -n ${SPOKE_NAMESPACE} -o json | jq -c '.items[].meta
 fi
 
 echo "Destroy the hosted cluster"
-hypershift destroy cluster agent --name $ASSISTED_CLUSTER_NAME --namespace $SPOKE_NAMESPACE --cluster-grace-period 60m
+hypershift_cli destroy cluster agent --name $ASSISTED_CLUSTER_NAME --namespace $SPOKE_NAMESPACE --cluster-grace-period 60m
```

Comment:
Do we need to clean up all the stuff we mirrored?

Reply:
I don't believe so, this is usually run in an ephemeral pod which gets removed at the end of a CI test. 

Though do let me know if that's not the case and we do need to add in clean up for the images!